force-app/main/default/classes/SOQL.cls

```diff
@@ -85,9 +107,22 @@ public with sharing class SOQL {
      * @return A map of Task Ids to Task records.
      * Update the corresponding test class and testGetTasksDueNextWeek method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static Map<Id, Task> getTasksDueNextWeek() {
+    Map <Id, Task> idAndTaskMap = new Map <Id, Task>();
+    Date today = Date.today();
+    Date nextWeek = today.addDays(7);
+    List <Task> tasksDue = [
+        SELECT Id, Subject, ActivityDate 
+        FROM Task 
+        WHERE ActivityDate >= :today 
+        AND ActivityDate <= :nextWeek
+        ];
```

Comment:
You can use Date variables and binds for this. There are also relative date formula you can use in queries. Similar to the ones you can use in reports.